README.md

```diff
@@ -24,21 +24,20 @@ import (
 
 func send() {
         client := courier.CourierClient("<AUTH_TOKEN>")
-        var message = []byte(`{
-                eventId: "<EVENT_ID>", // get from the Courier UI
-                recipientId: "<RECIPIENT_ID>"
-                profile: {
-                        email: "example@example.com",
-                        phone_number: "555-228-3890"
-                },
-        data: {} // optional variables for merging into templates
-        overrides: {} // optional http provider overrides
-        }`)
-        response, err := client.Send(message)
-        if err != nil {
-                log.Fatalln(err)
+
+        myData := struct {
+                foo string
+        }{
+                foo: "bar",
         }
-        log.Println(response.MessageId)
+        myProfile := struct {
```

Comment:
small change request: `myProfile` => `profile`. reason being is that `profile` is request-scoped for recipients (not "mine").